### ISSUE TITLE
Collect client timings without showing the UI

### ISSUE THREAD

**Issue by dchaib** [MiniProfiler/dotnet#329]
On alpha versions of MiniProfiler v4, I used to collect client timings in production. I just disabled the UI when the request was not local to make sure the customers didn't see it.

I have upgraded to the stable version (4.0.133) but the client timings are no longer collected in production, it only works for local requests. I believe this is because of #239.

Am I correct? Is there a way to collect client timings in production without showing the UI to the visitors?

**Comment by NickCraver** [comment]
You're correct in this isn't *currently* supported (if a user can't see results, we don't render the JS...because that was resulting in a lot of needless script downloads for most apps).

What are you interested here? You want to enable the profiler server-side, not render results, and upload the client timings for storage on it? Or something else?

I'm not at all against enabling the use case in a better way, just want to make sure we're on the same page first!

**Comment by NickCraver** [comment]
We're doing server-side on Stack Overflow for all requests, but not using MiniProfiler to collect client timings (we do this through another mechanism at a much larger scale). The scenario helps though - I'll think about this one and how we can most simply accommodate the case.

**Comment by timhenstridge** [comment]
Hi,
I was interested in this idea also, my thoughts was that exposing even hidden server side details would be a security issue in production (user taps Alt+P) and I was looking at a way of recording useful client timings without exposing the extra server side timings, then I found this issue. 
Not exposing SQL in any request/response data would be a highly desirable feature of an enhancement like this.  
I suppose an  "upload client timings only" POST which could have a different permission rule would be a nice to have also.
Is there a way I could assist in developing a feature like this? (i'm looking for suggestions on where to start)
